test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/QueryExecution/ExecuteTests.cs

```diff
@@ -115,9 +115,10 @@ public void DatabaseChangesWhenCallingUseDatabase()
             Assert.True(multipleElapsedTime > elapsedTime);
         }
 
-        public Query CreateAndExecuteQuery(string queryText, ConnectionInfo connectionInfo, IFileStreamFactory fileStreamFactory)
+        public static Query CreateAndExecuteQuery(string queryText, ConnectionInfo connectionInfo, IFileStreamFactory fileStreamFactory, bool IsSqlCMD = false)
```

Comment:
nit: I think according to .Net naming conventions parameter should be named `IsSqlCmd`